ansible/pbTestScripts/testScript.sh

```diff
@@ -1,5 +1,9 @@
 #!/bin/bash
-set -eu
+set -u
+
+branchName='NULL'
+folderName=' '
```

Comment:
Best not to assign anything to initialize rather than a single whitespace string.